app/controllers/meetings_controller.rb

```diff
@@ -89,9 +90,9 @@ def leave
 
   # DELETE /meetings/1
   def destroy
-    redirect_unless_leader_for(@meeting.group)
+    redirect_unless_leader_for(@meeting.group) && return
     # Notify group members that the meeting has been deleted
-    send_notification(@meeting, group.members, 'remove_meeting')
+    send_notification(@meeting, @meeting.group.members, 'remove_meeting')
```

Comment:
I think this should be `group.group_members`
```suggestion
    send_notification(@meeting, group.group_members, 'remove_meeting')
```

Reply:
here I've added the `@meeting` first cause the variable group is not set up in this action and `@meeting` is and you can access the group through it.

And I've used `group.members` cause it sounded better than `group.group_members` and its a valid relationship defined in group model, is basically just a has_many throug group_members. 
https://github.com/ifmeorg/ifme/blob/master/app/models/group.rb#L20